tutorials/shading/shading_reference/canvas_item_shader.rst

```diff
@@ -48,11 +48,16 @@ Global built-ins
 
 Global built-ins are available everywhere, including custom functions.
 
-+-------------------+--------------------------+
-| Built-in          | Description              |
-+===================+==========================+
-| in float **TIME** | Global time, in seconds. |
-+-------------------+--------------------------+
++-----------------------+-----------------------------------------------------------------------------+
+| Built-in              | Description                                                                 |
++=======================+=============================================================================+
+| in float **TIME**     | Global time, in seconds.                                                    |
+|                       | It's subject to the rollover setting (which is 3,600 -1 hour- by default).  |
++-----------------------+-----------------------------------------------------------------------------+
+| in vec4 **MODULATE**  | Final modulate color.                                                       |
```

Comment:
The way you have defined ``MODULATE`` it is not a global built-in, it is a built in that is available in all three shader functions. Global built-ins are a special type that is accessible everywhere, including custom functions. They are only available in master, and are specified with "global" keyword. see https://github.com/godotengine/godot/pull/37166

Reply:
Understood.

But now I'm confused about  _master_ in this repo being for 3.2. Is that no longer true? If that's the case, I guess I can leave the changes to TIME here and move on MODULATE to a separate PR for 3.2.

Reply:
I think master in this repo is for 4.0 because there is a separate 3.2 branch. So yea, maybe leave this PR with the time changes and then make another PR with the modulate changes for 3.2.

Reply:
Yeah, `master` here is for 4.0.

Reply:
PR updated to include only the TIME stuff.

Reply:
BTW, the PR template says that _master_ is fo 3.0. Maybe it needs an update.